gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -2353,13 +2378,13 @@ properties:
     $ref:
       - "_terms.yaml#/last_known_disease_status/common"
     enum:
+      - Biochemical evidence of disease without structural correlate
       - Distant met recurrence/progression
       - Loco-regional recurrence/progression
-      - Biochemical evidence of disease without structural correlate
       - Tumor free
       - Unknown tumor status
       - With tumor
-      - not reported
```

Comment:
Have we checked the migration status for this one?  Are these values already populated?